swagger_server/controllers/cryptography/keys.py

```diff
@@ -0,0 +1,89 @@
+TEST_PRIVATE_SIGNING_KEY = """-----BEGIN ENCRYPTED PRIVATE KEY-----
```

Comment:
Yep, I had to stick something in there whilst we have no plan on key management. They will be removed soon hopefully